ansible/roles/ocp4_workload_rhacm/tasks/pre_workload.yml

```diff
@@ -1,5 +1,45 @@
 ---
 # Implement your Pre Workload deployment tasks here
+- set_fact:
+    ansible_python_interpreter: "/bin/python"
```

Comment:
Why is this necessary ?

in rhel8 it's different IIRC, so that would probably not work there.
i'd rather let ansible detect the default python interpreter.

Reply:
If you don't put that, the first run will work, but the next run it will say you need to install python2 bindings for the pip module to work.

Reply:
@mvazquezc This should really be done by the workload deployer itself. We have the k8s environment on the hosts. 

E.g. https://github.com/redhat-cop/agnosticd/blob/development/ansible/configs/ocp4-cluster/workloads.yml#L11
or https://github.com/redhat-cop/agnosticd/blob/development/ansible/configs/ocp4-workshop/ocp_workloads.yml#L13

or even: https://github.com/redhat-cop/agnosticd/blob/development/ansible/configs/ocp-workloads/ocp-workload.yml#L11

Reply:
I tried with a different lab from opentlc and it seems it works, I cannot fully validate the workload since I need 4.4 cluster. (I was using the installation lab for testing, that's why I needed to setup the virtualenv).

Reply:
Ah. Makes sense.